src/commands/resetdir.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    SimbaConfig,
+    AllDirs,
+} from "@simbachain/web3-suites";
+import {default as chalk} from 'chalk';
+import yargs from 'yargs';
+
+export const command = 'resetdir';
+export const describe = 'reset default path to directory for "build", "artifact", "artifacts", "contract", or "contracts"';
+export const builder = {
+    'dirname': {
+        'string': true,
+        'type': 'string',
+        'describe': '"build" or "contract" or "contracts" or "artifact" or "artifacts" or "all"',
```

Comment:
Could use a choice type that would validate the entry. --savemode for export command has this option.

Reply:
fixed